src/tests/java8/java.util.functionTest/ClockTest.java

```diff
@@ -0,0 +1,64 @@
+package java8;
+import org.junit.Test;
+import gov.nasa.jpf.util.test.TestJPF;
+import org.junit.Test;
+import java.time.*;
+import java.time.temporal.ChronoUnit;
+import java.util.function.Function;
+import static org.junit.Assert.*;
+import static org.hamcrest.CoreMatchers.*;
+
+
+public class ClockTest {
+
+
+     private Function<String,ZoneId> withZoneId= (timeZoneId -> ZoneId.of(ZoneId.SHORT_IDS.get(timeZoneId)));
+
+
+    @Test
+    public void fixed_clock_test() throws Exception{
+        Instant fixedInstant  = Instant.parse("2014-03-06T21:27:31Z");
+        Clock fixedClock = Clock.fixed(fixedInstant, withZoneId.apply("EST"));
+
+        assertThat(fixedClock.instant(),is(fixedInstant));
+        Thread.sleep(3000l);
```

Comment:
Thread.sleep does not work in the same way in JPF as normal; in this case, the test still makes sense, though.